src/Tribe/Attendees.php

```diff
@@ -320,9 +320,19 @@ public function enqueue_assets( $hook ) {
 				'check'     => wp_create_nonce( 'move_tickets' ),
 				'TB_iframe' => 'true',
 			) ),
+			'confirmation'  => esc_html__( 'Please confirm that you would like to delete this attendee.', 'event-tickets' ),
```

Comment:
:warning: Array double arrow not aligned correctly; expected 4 space(s) between "'confirmation'" and double arrow, but found 2.

<sub>hash: 52ed871a67a9d669e8b2155fd81d1eb0</sub>